dev/NavigationView/NavigationView.xaml

```diff
@@ -75,6 +75,16 @@
                                 </VisualState>
                             </VisualStateGroup>
 
+                            <VisualStateGroup x:Name="PaneOpenStateGroup">
```

Comment:
>PaneOpenStateGroup [](start = 54, length = 18)

It feels weird to have the hamburger/Close/back buttons moving. 
I think it would be better to always keep them aligned with the other icons.

Reply:
I thought about that too, and I think we would need to set the width of the buttons manually. Setting their width to auto does not work, and removing the set width from the "BackButton" template results in a crash for some reason.

Would this "always" aligned also apply to displaymode LeftMinimal or should we keep their default width (40) when in LeftMinimal regardless of CompactPaneLength?